### DISCUSSION TITLE
Newline in custom report template

### DISCUSSION THREAD

**Discussion by wouter2397** [aquasecurity/trivy#2294] [Q&A] [answered]
Hello all,

I'm trying to export Trivy container vulnerability data back to GitLab (We don't use GitLab CI).
The problem is that the original table output is to long for a GitLab comment and therefor will be unreadable.

Now I'm trying to create a smaller template by defining only the required field I want.
Only problem is that I don't know how to create a newline within the template.

It probably is something very easy..... Is there someone who can help me with this?

**Comment by liamg** [comment, marked as answer]
Hi @wouter2397, you should be able to use a literal new line character, no need for `\n` or anything special. You can see some template examples here: https://github.com/aquasecurity/trivy/tree/main/contrib

**Reply by wouter2397** [reply]
@liamg Thanks for your response. The suggestion you proposed indeed does the trick. Problem for was I was working with an inline template within the trivy command, but as you pointed out it's best to create a seperate template file for this.

After I did this everything working like I wanted.
Thanks again for the quick reply that solved my problem!
